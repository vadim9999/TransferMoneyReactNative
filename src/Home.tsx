import React from 'react';
import { StyleSheet, Text, View, StatusBar } from 'react-native';


class Home extends React.Component<{},{}>{

  render(){

    return(
      <View>
        <Text>ok1</Text>
      <Text>
        Далі
      </Text>
      {/* <StatusBar barStyle = "dark-content" hidden = {false} backgroundColor = "#00BCD4" translucent = {true}/>
  <View>
    
    <Text>Okk</Text>
    </View> */}
</View>
    )
    
  }
}

export default Home
// const styles = StyleSheet.create({
//   container: {
//     flex: 1,
//     backgroundColor: '#fff',
//     alignItems: 'center',
//     justifyContent: 'center',
//   },
// });
