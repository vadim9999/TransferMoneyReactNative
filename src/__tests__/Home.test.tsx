
// import Adapter from 'enzyme-adapter-react-16'
import React from 'react'
// import Enzyme, {mount} from 'enzyme'
import Home from '../Home'
import renderer from 'react-test-renderer';
// Enzyme.configure({adapter: new Adapter})
import {render, fireEvent} from 'react-native-testing-library'

describe('Testing ',()=>{

  let wrapper;

  beforeEach(()=>{

    // wrapper = mount(<Home />)
  })

  it('testing button on click', ()=>{
    const {getByText} = render(<Home />)
    // const button = getByText('Далі')

    // fireEvent.press(button)
    // console.log(button);
  })

  it.skip('test renders correctly',()=>{
    
    
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot()
    
    console.log("this is the first test");
    
  })
})