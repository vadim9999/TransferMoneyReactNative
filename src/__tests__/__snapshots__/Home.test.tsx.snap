// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing  test renders correctly 1`] = `
<View>
  <Text>
    Ok
  </Text>
</View>
`;
